api/src/main/java/org/openmrs/api/PatientService.java

```diff
@@ -106,6 +106,17 @@ public interface PatientService extends OpenmrsService {
 	@Authorized( { PrivilegeConstants.GET_PATIENTS })
 	public Patient getPatientByUuid(String uuid) throws APIException;
 	
+	/**
+	 * Get patient by universally unique identifier.
```

Comment:
should it not then be like  
`Returns a lists of patients using a unique identifier`

Reply:
I think this being an abstract method, using `Get patient ...` in the java doc is pretty fine, then we can use `Returns a list of patient ...` within them method implementation.